### DISCUSSION TITLE
Input type "link"

### DISCUSSION THREAD

**Discussion by fuelkoy** [pdfme/pdfme#81] [Ideas]
(First of all this library looks, feels and works great. Just found and really well pleased using it)

In short: is there a way or should there be a way to add custom link to template?

In the base pdf there of course can be links but what about providing custom link to template?
And if it is not possible would implementing this kind of functionality be good and usable?

For example Google maps has service where coordinates are passed to predefined url. The result link will show the given coordinates in Google maps. These coordinates regarding of usage can change every time, so static link in pdf doesn't work.

In this case input with given text and url would provide a functionality to create a custom link.

**Comment by hand-dot** [comment]
Hi @fuelkoy 

Sorry for the late reply.
Unfortunately, I don't have a plan to add the Input type "link".

**Comment by karstenwinter** [comment]
I'd also be interested in a link element that has different display text than url content would be nice

Thanks that URLs work already (as long as they are on one line :D https://github.com/pdfme/pdfme/issues/73)

A solution you'd like:
something like `<a href="url">text</a>` would be ideal :)

alternatives I've considered:
shortlinks :)

A clickable image that opens a URL would also work :)
